# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lcdutils/makefile-useprefix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Makefile.orig	2004-01-30 19:04:40.000000000 +0100
+++ ./Makefile	2004-01-30 19:04:17.000000000 +0100
@@ -2,12 +2,13 @@
 #LDFLAGS=-static
 
 CFLAGS=-O2 -Wall
+PREFIX=/usr/local
 
 all: putlcd buttond
 
 install: all
-	install putlcd /usr/local/sbin
-	install buttond /usr/local/sbin
+	install putlcd ${PREFIX}/sbin
+	install buttond ${PREFIX}/sbin
 
 clean:
 	rm -f putlcd buttond *.o
